R/parameters.R

```diff
@@ -289,3 +297,51 @@ identical_names <- function(x, y) {
 
   identical(x_names, y_names)
 }
+
+# adapted from knitr::combine_words
+combine_words <- function(words, sep = ", ", and = " and ", before = "", after = before,
```

Comment:
This code is copied from knitr's source so we don't have to take it on as a hard dependency. I wasn't able to find any r-lib/other dependencies that re-export this code or something like it, though it seems like it would be widely useful in prompting, generally.

Reply:
Since dials is aleady depending o glue you should use `glue::glue_collapse()`, the following gives the same 
``` r
knitr::combine_words(letters)
glue::glue_collapse(letters, ", ", last = ", and ")
```

Reply:
Ah, perfect! Thanks.

Reply:
Addressed in ed8dd18.